scripts/ci-build-kubernetes.sh

```diff
@@ -78,13 +75,15 @@ setup() {
     # ref: https://github.com/kubernetes/kubernetes/blob/5491484aa91fd09a01a68042e7674bc24d42687a/build/lib/release.sh#L345-L346
     export KUBE_IMAGE_TAG="${KUBE_GIT_VERSION/+/_}"
     echo "using K8s KUBE_IMAGE_TAG=${KUBE_IMAGE_TAG}"
+
+    export AZURE_BLOB_CONTAINER_NAME="${AZURE_BLOB_CONTAINER_NAME:-"kubernetes-ci"}"
```

Comment:
@marosset so I realized that since we were setting the same env var in both scripts to different values, we were overwriting the value 

For now the easiest way forward is to use the same container name in both scripts (and not overwrite the var if something outside already set it so it can be configured). In the future I'd like to split up the ccm binaries into a different path in the container so it's clear what's what to humans (right now it's just commit SHAs) but that would require another change to the CCM templates so let's keep it as-is for now.

Reply:
sounds good to me